src/sentry/static/sentry/app/actions/projectActions.jsx

```diff
@@ -20,4 +20,5 @@ export default Reflux.createActions([
   'removeTeam',
   'removeTeamSuccess',
   'removeTeamError',
+  'loadProjects',
```

Comment:
Nit: Might be better to move this to Line 8-11, or even rearrange all the constants to alphabetical order.

Reply:
Relevant for `teamActions.jsx`, `projectsStore.jsx`, `teamStore.jsx` too.